src/main/java/bookstore/dto/book/BookDto.java

```diff
@@ -1,13 +1,19 @@
 package bookstore.dto.book;
 
 import java.math.BigDecimal;
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
 
-public record BookDto(
-        Long id,
-        String title,
-        String author,
-        String isbn,
-        BigDecimal price,
-        String description,
-        String coverImage) {
+@Setter
+@Getter
+public class BookDto {
```

Comment:
why not record?

Reply:
Because I need a setter in BookMapper for set CategoryIds